src/osd/modules/sound/js_sound.js

```diff
@@ -174,9 +174,10 @@ function tick (event) {
 		}
 	}
 	//Pad with silence if we're underrunning:
```

Comment:
This comment is no longer accurate, you're now padding with the previous sample and not silence